docs/en/observability/monitor-servers.asciidoc

```diff
@@ -32,3 +32,115 @@ This metric relies on the same indices as the logs.
 
 For information about which required fields the {metrics-app} uses to display host metrics, see the
 <<metrics-app-fields,Metrics field reference>>.
+
+[[enhanced-host-metrics]]
+== Enhanced host metrics
+
+Without leaving the <<view-infrastructure-metrics,*Inventory*>> page, you can view enhanced metrics relating to each host
```

Comment:
Host details provide more than just metrics, so it should say something like. "you can view details relating..."

Reply:
Thanks.